docs/src/app/components/PropTypeDescription.jsx

```diff
@@ -51,7 +53,9 @@ const PropTypeDescription = React.createClass({
     }
 
     return (
-      <MarkdownElement text={text} />
+      <div className="propsTable">
```

Comment:
I'm gonna be rude, but I think that `PropTypeDescription` or in lower case would be a more explicit className. Though ?